fbw-a32nx/src/wasm/fadec_a320/src/EngineControl.h

```diff
@@ -214,15 +222,15 @@ class EngineControl {
       if (engineState == 0 || engineState == 10) {
         if (engineIgniter == 1 && engineStarter == 1 && simN2 > 20) {
           engineState = 1;
-        } else if (engineIgniter == 2 && engineStarter == 1) {
+        } else if (engineIgniter == 2 && engineMasterTurnedOn) {
           engineState = 2;
         } else {
           engineState = 0;
         }
       }
 
       // Present State ON
-      if (engineState == 1 || engineState == 11) {
+      else if (engineState == 1 || engineState == 11) {
```

Comment:
If you're gonna use `else if` here, I suggest moving this line to the closing brace above.